pkg/models/network.go

```diff
@@ -32,6 +33,8 @@ func (nk NetworkKind) String() string {
 		return "Local Network"
 	case Devnet:
 		return "Devnet"
+	case Etna:
```

Comment:
pls wait for https://github.com/ava-labs/avalanche-cli/pull/2233 to be merged as this PR implements this in a different way